lib/install/validate/storage.go

```diff
@@ -32,14 +33,26 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 	defer trace.End(trace.Begin(""))
 
 	// Image Store
-	imageDSpath, ds, err := v.DatastoreHelper(ctx, input.ImageDatastoreName, "", "--image-datastore")
+	imageDSpath, ds, err := v.DatastoreHelper(ctx, input.ImageDatastorePath, "", "--image-datastore")
+
+	// provide a default path if only a DS name is provided
+	if imageDSpath.Path == "" {
+		imageDSpath.Path = path.Join(input.DisplayName, "images")
+	}
+
 	v.NoteIssue(err)
 	if ds != nil {
 		v.SetDatastore(ds, imageDSpath)
 		conf.AddImageStore(imageDSpath)
 	}
 
 	// Volume Store(s)
+	conf.BootstrapImagePath = fmt.Sprintf("[%s] %s/%s",
+		imageDSpath.Host,
+		input.DisplayName,
```

Comment:
I found where it is. Thanks.
Then unfortunately, this problem exists before. The correct file path is set here:
https://github.com/vmware/vic/blob/master/lib/install/management/appliance.go#L414

Reference here https://github.com/vmware/vic/blob/master/lib/install/management/appliance.go#L362 for how to set correct file path, bootstrap.iso is located in the same place with appliance.iso.